ravenpy/config/rvs.py

```diff
@@ -0,0 +1,913 @@
+        else:
+            raise ValueError(f"Value {v} should be one of {RVI.EVAPORATION_OPTIONS}.")
+
+    @property
+    def ow_evaporation(self):
+        """Open-water evaporation scheme"""
+        return self._ow_evaporation
+
+    @ow_evaporation.setter
+    def ow_evaporation(self, value):
+        v = value.upper()
+        if v in RVI.EVAPORATION_OPTIONS:
+            self._ow_evaporation = v
+        else:
+            raise ValueError(f"Value {v} should be one of {RVI.EVAPORATION_OPTIONS}.")
+
+    @property
+    def calendar(self):
+        """Calendar"""
+        return self._calendar.upper()
+
+    @calendar.setter
+    def calendar(self, value):
+        if value.upper() in RVI.CALENDAR_OPTIONS:
+            self._calendar = value
+        else:
+            raise ValueError(f"Value should be one of {RVI.CALENDAR_OPTIONS}.")
+
+    def _dt2cf(self, date):
+        """Convert datetime to cftime datetime."""
+        return cftime._cftime.DATE_TYPES[self._calendar.lower()](*date.timetuple()[:6])
+
+    def to_rv(self):
+
+        # Attributes
+        a = list(filter(lambda x: not x.startswith("_"), self.__dict__))
+
+        # Properties
+        p = list(
+            filter(
+                lambda x: isinstance(getattr(self.__class__, x, None), property),
+                dir(self),
+            )
+        )
+
+        d = {attr: getattr(self, attr) for attr in a + p}
+
+        return dedent(self.tmpl).format(**d)
+
+
+##########
+# R V P  #
+##########
+
+
+class RVP(RV):
+
+    tmpl = """
+    """
+
+    def __init__(self, config):
+        super().__init__(config)
+
+        # Model specific params and derived params
+        self.params = None
+        self.derived_params = None
+
+        self.soil_classes: Tuple[SoilClassesCommand.Record, ...] = ()
+        self.soil_profiles: Tuple[SoilProfilesCommand.Record, ...] = ()
+        self.vegetation_classes: Tuple[VegetationClassesCommand.Record, ...] = ()
+        self.land_use_classes: Tuple[LandUseClassesCommand.Record, ...] = ()
+        self.channel_profiles: Tuple[ChannelProfileCommand, ...] = ()
+        self.avg_annual_runoff: Optional[float] = None
+
+    def to_rv(self):
+        d = {
+            "params": self.params,
+            "derived_params": self.derived_params,
+            "soil_classes": SoilClassesCommand(self.soil_classes),
+            "soil_profiles": SoilProfilesCommand(self.soil_profiles),
+            "vegetation_classes": VegetationClassesCommand(self.vegetation_classes),
+            "land_use_classes": LandUseClassesCommand(self.land_use_classes),
+            "channel_profiles": "\n\n".join(map(str, self.channel_profiles)),
+            "avg_annual_runoff": AvgAnnualRunoffCommand(self.avg_annual_runoff),
+        }
+        return dedent(self.tmpl).format(**d)
+
+
+#########
+# R V T #
+#########
+
+
+class RVT(RV):
+
+    tmpl = """
+    {gauge}
+
+    {forcing_list}
+
+    {observed_data}
+    """
+
+    NC_VARS = {
```

Comment:
Maybe would make more sense to key by Raven name (TEMP_MIN), and include the standard_name (tasmin) as the "standard_name" attribute. Minor point, feel free to dismiss.

Reply:
J'ai essayé de changer les clés mais ça introduisait de la gymnastique plus compliquée pour la gestion de `RVT._var_cmds`, étant donné que les clés manipulées par l'usager sont les standard_names.. Je me suis donc contenté de faire un peu de ménage dans la structure de données, et de changer qq noms de variables.